### PR TITLE
Migrate portable zip to ipex-llm

### PR DESCRIPTION
**Pull request by JinBridger**

### CODE REVIEW

**Review by Oscilloscope98 — COMMENTED** [review]
@gc-fu Please have a look at the change in chat-ui.bat :) Thank you

**Review by gc-fu — APPROVED** [review]
I am not familiar with the windows things, but the command `.\python-embed\python.exe -m ipex_llm.serving.fastchat.ipex_llm_worker --model-path %modelpath% --device cpu --low-bit "sym_int4" --trust-remote-code` LGTM

File: python/llm/portable-zip/chat-ui.bat [hunk 1]
```diff
@@ -11,7 +11,7 @@ if errorlevel 1 (
 )
 echo [1/3] Controller started successfully
 
-powershell -Command "Start-Process -FilePath PowerShell -ArgumentList '-Command', '& { .\python-embed\python.exe -m ipex_llm.serving.model_worker --model-path %modelpath% --device cpu > zip_model_worker.log 2>&1 }' -NoNewWindow"
+powershell -Command "Start-Process -FilePath PowerShell -ArgumentList '-Command', '& { .\python-embed\python.exe -m ipex_llm.serving.fastchat.ipex_llm_worker --model-path %modelpath% --device cpu --low-bit "sym_int4" --trust-remote-code > zip_model_worker.log 2>&1 }' -NoNewWindow"
 timeout /t 1 /nobreak >nul 2>&1
```
Comment (Oscilloscope98) [inline python/llm/portable-zip/chat-ui.bat:15]: Do we need to set `no_proxy` for chat-ui.bat?

Comment (gc-fu) [reply python/llm/portable-zip/chat-ui.bat:15]: Yes, probably you will need no_proxy for worker to register itself to controller

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**